core-backend/sandbox/Cargo.toml

```diff
@@ -12,7 +12,8 @@ gear-backend-common = { path = "../common" }
 
 parity-wasm = { version = "0.45.0", default-features = false }
 sp-sandbox = { version = "0.10.0-dev", git = "https://github.com/gear-tech/substrate.git", branch = "gear-stable", default-features = false, features = ["host-sandbox"] }
-log = { version = "0.4.17", default-features = false }
+# Use max_level_debug feature to remove tracing in sys-calls by default.
+log = { version = "0.4.17", default-features = false, features = ["max_level_debug"] }
```

Comment:
Despite the fact, that this won't affect log output, it still contains overhead of single comparison:
https://github.com/rust-lang/log/blob/d0a45aa2929365c840b82360370b223d0c22364e/src/macros.rs#L35

I'd suggest to add feature to sandbox like "deep-log"

And use tracing logging inside the module under `#[cfg(feature = "deep-log")]` macro to cut it in compile time

Reply:
```rust
 if lvl <= $crate::STATIC_MAX_LEVEL
```
is comparison between const and const, which will be removed by any compiler